### PR TITLE
[Prow CI Workflow Enablement] add aws-ovn-hypershift-guest workflow and guest e2e test

### PR DESCRIPTION
**Pull request by LiangquanLi930**
hypershift-guest-workflow, guest e2e test
https://issues.redhat.com/browse/OCPQE-12433

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -111,6 +111,18 @@ tests:
...
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: e2e-aws-ipi-ovn-hypershift-guest-p2
+  cron: 0 5 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_ADDITIONAL: Hypershift
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:121]: excludes disruptive cases?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -164,6 +164,18 @@ tests:
...
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: e2e-aws-ipi-ovn-hypershift-guest-p2
+  cron: 0 6 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_ADDITIONAL: Hypershift
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:174]: excludes disruptive cases?

**Comment by heliubj18** [comment]
LGTM

**Comment by LiangquanLi930** [comment]
@kuiwang02  Hi, PTAL this PR when you have time? thanks

**Comment by kuiwang02** [comment]
@LiangquanLi930 per https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/33118/rehearse-33118-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-aws-ipi-ovn-hypershift-guest-p2/1580778439382667264/artifacts/e2e-aws-ipi-ovn-hypershift-guest-p2/cucushift-hypershift-extended-enable-guest/build-log.txt, there is passwd in the log.

it is from the code https://github.com/openshift/release/pull/33105/files#diff-3dc982615dd4491a38e200780ba90cc646a0ef6c5cf72bdfdcc883fb65832589R16 of the PR 33105.
it is not allowed.

I guess you print it because there is such passwd in jenkins job. you known, jenkins server is internal, but here is public.
So, please remove these four lines which are printing passwd. please do not output sensitive information.

**Comment by kuiwang02** [comment]
note: because your enable debug with `set -x`, the https://github.com/openshift/release/pull/33105/files#diff-3dc982615dd4491a38e200780ba90cc646a0ef6c5cf72bdfdcc883fb65832589R13-R14 will also output passwd.
so, please disable shell debug.

**Comment by LiangquanLi930** [comment]
> note: because your enable debug with `set -x`, the https://github.com/openshift/release/pull/33105/files#diff-3dc982615dd4491a38e200780ba90cc646a0ef6c5cf72bdfdcc883fb65832589R13-R14 will also output passwd. so, please disable shell debug.

https://github.com/openshift/release/pull/33120 done, thanks for your suggest

**Comment by kuiwang02** [comment]
I am OK now

**Comment by LiangquanLi930** [comment]
@kuiwang02 could you help add lgtm now? This PR doesn't seem to need anyone else's review.

**Comment by kuiwang02** [comment]
once I add lgtm, it will be merged.  but it need productivity team to review and approve it.
so, I say `I am OK` from my view.

**Comment by LiangquanLi930** [comment]
@liangxia could you help to review this PR, thanks

**Pull request merged**